docs/sources/reference/run.md

```diff
@@ -552,6 +553,20 @@ would be 2*300M, so processes can use 300M swap memory as well.
 We set both memory and swap memory, so the processes in the container can use
 300M memory and 700M swap memory.
 
+We can set oom control in which whether to disable OOM Killer or not when OOM
+happened. It is strongly recommended that use this option with `-m`.
+
+Examples:
+
+    $ docker run -ti -m 100M oom-kill-disable=true ubuntu:14.04 /bin/bash
```

Comment:
Missing `--` before the flag/option. Also, for Boolean flags, the `true` can be omitted, so just `--oom-kill-disable`